docs/src/gs/create-workload-cluster.md

```diff
@@ -6,41 +6,47 @@ Choose one of the available templates for to create your workload clusters from
+| `OCI_CONTROL_PLANE_SHAPE`                 | VM.Standard.E4.Flex | The shape of the Kubernetes nodes                                                                                                                                                                                                                                                                                                                                                                            |
+| `OCI_CONTROL_PLANE_SHAPE_OCPUS`           | 1                   | The number of OCPUs allocated to the control plane instance                                                                                                                                                                                                                                                                                                                                                  |
+| `OCI_WORKER_SHAPE`                        | VM.Standard.E4.Flex | The shape of the Kubernetes worker nodes                                                                                                                                                                                                                                                                                                                                                                     |
+| `OCI_WORKER_SHAPE_OCPUS`                  | 1                   | The number of OCPUs allocated to the worker instance                                                                                                                                                                                                                                                                                                                                                         |
+| `OCI_SSH_KEY`                             |                     | The public SSH key to be added to the Kubernetes nodes. It can be used to login to the node and troubleshoot failures.                                                                                                                                                                                                                                                                                       |
+| `OCI_CONTROL_PLANE_PV_TRANSIT_ENCRYPTION` | true                | [In-transit encryption](https://docs.oracle.<br/>com/en-us/iaas/Content/File/Tasks/intransitencryption.htm) provides a way to secure your data between instances and mounted file systems using TLS 1.2 (Transport Layer Security) encryption. Only [some bare metal instances](https://docs.oracle.com/en-us/iaas/releasenotes/changes/60d602f5-abb3-4639-aa19-292a5744a808/) support in-transit encryption |
```

Comment:
```suggestion
| `OCI_CONTROL_PLANE_PV_TRANSIT_ENCRYPTION` | true                | Enables [in-flight Transport Layer Security (TLS) 1.2 encryption](https://docs.oracle.com/en-us/iaas/Content/File/Tasks/intransitencryption.htm) of data between control plane nodes and their associated block storage devices |
```

I would then add a footnote to this: "Note that only [specific bare metal shapes](https://docs.oracle.com/en-us/iaas/releasenotes/changes/60d602f5-abb3-4639-aa19-292a5744a808/) support in-transit encryption. If an unsupported shape is specified, the deployment will..."

And then add either "fail completely" or "succeed with encryption disabled" based on the outcome of either an existing or future test to determine what the actual outcome is. I would suggest we go with pre-flight validation of the chosen shape with selected features and fast-fail on a mismatch.